From: Margarita Manterola <marga@debian.org>
Date: Wed, 4 Apr 2018 14:00:29 +0200
Subject: Use gtk3 engine when using QT5

Qt base recommends the gtk3 platformtheme, while qt5ct provides an Xsession
script that sets QT_QPA_PLATFORMTHEME correspondingly. So we set
QT_QPA_PLATFORMTHEME to gtk3, if not already set.
---
 cinnamon-session/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cinnamon-session/main.c b/cinnamon-session/main.c
index 800a622..089aa02 100644
--- a/cinnamon-session/main.c
+++ b/cinnamon-session/main.c
@@ -344,7 +344,7 @@
         /* Make QT5 apps follow the GTK style. Starting with QT 5.7, a different
          * env var has to be set than what worked in previous versions.
          */
-        qt_platform_theme_new = HAVE_QT57 ? "qt5ct" : "qgnomeplatform";
+        qt_platform_theme_new = HAVE_QT57 ? "gtk3" : "qgnomeplatform";
 
         if (NULL == g_getenv ("QT_QPA_PLATFORMTHEME")) {
             csm_util_setenv ("QT_QPA_PLATFORMTHEME", qt_platform_theme_new);
